saleor/account/models.py

```diff
@@ -160,6 +160,21 @@ def get_ajax_label(self):
                 address.first_name, address.last_name, self.email)
         return self.email
 
+    def get_addresses(self):
+        # Set default shipping/billing address pk to -1
+        # if default shipping/billing address doesn't exist
+        default_shipping_address_pk, default_billing_address_pk = -1, -1
```

Comment:
Is it possible to use `None` instead of `-1` here?

Reply:
Yeah it can be set to None in grammar, but then it will be harder to tell it's because user doesn't have default address or backend doesn't annotate. For now, only ```resolve_addresses``` have ```is_default_shipping/billing_address``` rather than ```resolve_default_shipping/billing_address```. I think it's better to set ```-1``` (```is_default_shipping/billing_address``` will be False instead of None) to indicate the addresses are annotated but user doesn't have default address.

Reply:
I don't know if is a problem if we returned `True/False` when we have information whether an address is the default one or `null` if we don't know. Frontend client would care about the `True` value anyway.

Reply:
That's true for the frontend. But I think we could still keep None and False both as falsy cases since it could tell more information? If we treat unknown state as False, technically it could be True in some cases but we just don't know the information.

Reply:
So maybe I misunderstood you - I'd prefer to have here three possible values: true/false/null. I only don't like setting `-1` because it's kind of a "magic number" which I'd expected in pieces of code where we have some iteration or operate on counters etc. In terms of Django and model related stuff `null` just seems to me more appropriate.

Reply:
Yeah I agree. I just can't find a better approach. Setting it as ```None``` is just same as the query is not annotated. That's why I set it to ```-1``` as a special case.

Maybe I could define -1 as const with meaningful name?

Reply:
Never mind, i'm silly. There is a difference between variable as None and variable doesn't exist for ```getattr```. I will refactor it tomorrow.